### ISSUE TITLE
Add the date of a answer as context to the dataset

### ISSUE THREAD

**Issue by stefangrotz** [LAION-AI/Open-Assistant#2758]
I see quite often that answers are heavily dependent on the date, but people don't include that into their answers.

For Example:

Question:
> How old is Barack Obama?

Answer: 
> BO is 61 years old ....

It would be good to add the date of the answer as metadata to it wouldn't it? What is the data format of the finished Dataset?

**Comment by CheckMC** [comment]
good idea to add this to the metadata if it isn't already / if the date and time of answers are stored

**Comment by atemiz** [comment]
I think it is useful to answer with the date for time-dependent information. For example, for age-related questions, the answer might be: "As of April 20, 2023, Barack Obama would be 61 years old."

In this way, when the current date information is given with the system message, the assistant can make the necessary calculation or make a request to the service that will make this calculation.

I don't know how well this works, but in my experiments with ChatGPT, when I give the current date information, it can update its answer.

Or we need to find a suitable way for the assistant to write the operations that need to be calculated into its response, such as javascript template literals. But in this case the answers may not be used in raw format.
